internal/driver/driver.go

```diff
@@ -56,6 +54,21 @@ func NewProtocolDriver() sdkModel.ProtocolDriver {
 	})
 	return driver
 }
+func createX509Template() x509.Certificate {
+	template := x509.Certificate{
+		SerialNumber: big.NewInt(2023),
+		Subject: pkix.Name{
+			Organization: []string{"Baader.com"},
```

Comment:
if it was possible to use these hardcoded values from config, it would be chef's kiss and I would very much like to do that.

Reply:
I agree on majids point, as we want to avoid magic numbers as well and if we need to change them, we would access them from one place and everybody would be able to do some with some comments attached to configs.

Reply:
done